src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -190,6 +201,19 @@ public void deleteJobs(List<Job> jobs) {
     jobExecutionService.deleteJobs(jobs);
   }
 
+  @Override
+  public InputStream downloadExportedFile(UUID jobId) throws Exception {
+    Job job = getJobEntity(jobId);
+    if (job.getFiles().isEmpty()) {
+      throw new Exception("The URL of the exported file is missing");
+    }
+    URL url = new URL(job.getFiles().get(0));
```

Comment:
Please add java doc that this method will download first file and you can specify in this java doc that for EDIFACT we expect only 1 file contained in this files array